examples/webxr_vr_teleport.html

```diff
@@ -104,8 +104,15 @@
 					this.userData.isSelecting = false;
 
 					if ( INTERSECTION ) {
+						// reuse the same base reference space and watch out if our session changed
+						
+						if ( !baseReferenceSpace || session !== renderer.xr.getSession() ) {
```

Comment:
Is this check really necessary considering that you reset `customReferenceSpace`to `null` in WebXRManager on every session start?

Reply:
I believe it is actually.  Here's the scenario this tries to address:

1. user enters VR session A
2. user teleports to location A, but before we do this we need to cache the original starting reference space of VR session A, because once we call setReferenceSpace, we'll never be able to get it again.
3. user teleports a second time, since we have the cached original starting  reference we can just make a new reference space again.
4. user exits VR.  Our example has no way of knowing about this right now, so that cached `baseReferenceFrame` is from a VR session A that is no longer valid.
5. User enters into VR session B and tries to teleport.  If we use baseReferenceFrame as it is we'll get errors about modifying a reference frame of a session longer longer valid.

We needed some way to indicate that the baseReferenceFrame was from a session that is different from the current session.  I do this by comparing the active sessions to know whether to re-cache the original baseReference frame of the new VR Session B.

Reply:
Can't we just listen to the renderer's `sessionstart` event instead of this check?

Reply:
Thats a good suggestion! Less brute force checking and informs the user about the sessionstart/end events.